CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Guide to Making Developer Contributions
+
+## Overview:
+
+A CSV with package name and version is uploaded by the user, 
+then the risk of the packages are assessed using `riskmetric`.
+[TODO this is shakey for me.... where is this function actually called... 
+I see where we write comments to the database, 
+and where we read fields FROM the database,
+but where and how are the database's risk metrics written?]
+
+The database (SQLite by default) is updated to include the user specified package 
+and its associated metrics. 
+
+These metrics are used to populate the UI of the application. 
+Each package only needs to be entered into the database once for the user
+to see its associated risk.
+
+## Package files and folder 
+
+[TODO order in which to introduce these files, 
+maybe also include a schematic of how the files are related]
+
+### Root
+- `loggit.json` [TODO: log of user collected data within the app?]
+- `setup.R` Load all libraries, install libraries that aren't included. Called in `app.R.`
+- `risk_assessment_app.db` Database connection used within `Modules/DB.R`
```

Comment:
We are removing the `risk_assessment_app.db` file from the repo. The file will be created when the app is run the first time or if the file already exists, the app will use it. I think it is still important to mention this file here.